ansible/centos-ci/ansible/roles/vagrant/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+# file: ansible/roles/vagrant/install.yml
+---
+
+# For some reason this task gets always skipped :/
+#- name: "Install Vagrant from SCL"
+#  include_role:
+#    name: smbambling.scl
```

Comment:
Is `smbambling.scl` actually on the `roles_path` when it gets skipped? (I'm currently looking at http://docs.ansible.com/ansible/playbooks_roles.html#roles as I consider how we might be able to address https://github.com/leapp-to/prototype/issues/14 )